includes/classes/blocks/class-bindings.php

```
@@ -308,15 +308,18 @@ public function render_itinerary_block( $block_content, $parsed_block, $block_ob
 			return $block_content;
 		}
 
-		$pattern = $block_content;
-		$group   = array();
+		$build_template = $block_content;
```

PHPMD (2.15.0)

The variable $build_template is not named in camelCase. (undefined)

(CamelCaseVariableName)